src/editor/mainEditor/material/ui/BasicMaterial.tsx

```diff
@@ -1,14 +1,33 @@
 import * as React from "react";
+import ColorPicker, { ColorPickerType } from '../../../components/ColorPicker'
```

Comment:
move ColorPicker to  src/editor/mainEditor/material/ui/component/

Reply:
ColorPicker can use not only material color picking but also such as camera clear color, lighting color, so I made it there

Reply:
yes, i know your idea, but now it's only used by material component.
if in the future it'll be used by other component, then move it at that time.

Reply:
sure, you got it.